app/src/main/java/com/example/marvelapp/presentation/characters/CharactersFragment.kt

```diff
@@ -26,12 +37,64 @@ class CharactersFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         initCharactersAdapter()
+        observeInitialLoadState()
+
+        lifecycleScope.launch {
+            viewLifecycleOwner.lifecycle.repeatOnLifecycle(Lifecycle.State.STARTED) {
+                viewModel.charactersPagingData("").collect { pagingData ->
+                    charactersAdapter.submitData(pagingData)
+                }
+            }
+        }
     }
 
     private fun initCharactersAdapter() {
+        charactersAdapter = CharactersAdapter()
         with(binding.rvCharacters) {
+            scrollToPosition(0)
```

Comment:
This is only necessary if we want the default behavior of the Recycler View to scroll up to the first position when switching between the bottom navigation fragments